frontend/src/scenes/project-homepage/WatchNextPanel.tsx

```diff
@@ -28,7 +32,7 @@ export function RecordingRow({ recording }: RecordingRowProps): JSX.Element {
     return (
         <ProjectHomePageCompactListItem
             title={asDisplay(recording.person)}
-            subtitle={`Recorded ${dayjs(recording.start_time).fromNow()}`}
+            subtitle={`activity score: ${recording.mouse_activity_count}`}
```

Comment:
wanted to keep this vague so went with "activity score"
thought it might sound exciting